build.sbt

```diff
@@ -3,6 +3,9 @@ organization in ThisBuild := "sample.helloworld"
 // the Scala version that will be used for cross-compiled libraries
 scalaVersion in ThisBuild := "2.11.7"
 
+// the ConductR Developer Sandbox version that will be used to start the ConductR cluster
+SandboxKeys.imageVersion in Global := "1.1.2"
```

Comment:
I don't think that we should include the version here - the plugin needs to prompt for it in order to encourage people to get hold of the latest version and register.